nixio/cmd/nixexplore.py

```diff
@@ -0,0 +1,659 @@
+import os
+import argparse
+import nixio as nix
+import numpy as np
+import glob
+import datetime as dt
+import sys
+import matplotlib.pyplot as plt
```

Comment:
Hadn't realised the implications of this in the previous reviews.  Do we want nixpy to depend on matplotlib?  If we're going to be adding plotting functionality, I think this makes the script do a bit too much.  One of the reasons we created the nixworks package, at least for me, was to avoid having dependencies like matplotlib and pandas in nixpy.  Instead, nixworks has different submodules which become available based on optional dependencies.